test/coverage/datamodel/UMessageBuilderTest.cpp

```diff
@@ -11,28 +11,424 @@
 
 #include <gtest/gtest.h>
 #include <up-cpp/datamodel/builder/UMessage.h>
+#include <up-cpp/datamodel/serializer/UUri.h>
+#include <up-cpp/datamodel/validator/UUri.h>
+#include <up-cpp/datamodel/validator/Uuid.h>
 
 namespace {
+using namespace uprotocol::v1;
+using namespace uprotocol::datamodel::serializer::uri;
+using namespace uprotocol::datamodel::validator::uri;
+using namespace uprotocol::datamodel::builder;
+using namespace uprotocol::datamodel::validator::uuid;
 
-class TestFixture : public testing::Test {
+class TestUMessageBuilder : public testing::Test {
 protected:
-	// Run once per TEST_F.
-	// Used to set up clean environments per test.
-	void SetUp() override {}
-	void TearDown() override {}
-
-	// Run once per execution of the test application.
-	// Used for setup of all tests. Has access to this instance.
-	TestFixture() = default;
-	~TestFixture() = default;
-
-	// Run once per execution of the test application.
-	// Used only for global setup outside of tests.
-	static void SetUpTestSuite() {}
-	static void TearDownTestSuite() {}
+    // Run once per TEST_F.
+    // Used to set up clean environments per test.
+    void SetUp() override {
```

Comment:
This is called for each test case, but the values do not change. Is this intentional?

Reply:
What would be a better way to do this kind of setup ?

Reply:
If the values are completely constant, then they can be `constexpr` variables. If they need to be set up once for the whole test run, then then can be set in `SetUpTestSuite()`. If using variables set at runtime by one of the test methods, then care should be taken that tests cannot alter the values (e.g. provide accessors or make sure they are const).

Reply:
Done